README.md

```diff
@@ -31,7 +31,7 @@ website](https://golang.org).
 
 The easiest way is via `go get`:
 
-    $ go get -u github.com/maxmind/mm-geofeed-verifier
+    $ go get -u github.com/maxmind/mm-geofeed-verifier/v2
```

Comment:
`go get` is [deprecated](https://go.dev/doc/go-get-install-deprecation).  This should probably be `go install github.com/maxmind/mm-geofeed-verifier@latest`